assignments/lambda-classes.js

```diff
@@ -1 +1,152 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(testingPartOne){
+        this.name = testingPartOne.name;
+        this.age = testingPartOne.age;
+        this.location = testingPartOne.location;
+        this.gender = testingPartOne.gender;
+    }
+    speak(){
+        return `Hello my name is ${this.name}, I am from ${this.location}.`;
+    }
+}
+
+class Instructor extends Person{
+    constructor(testingPartTwo){
+        super(testingPartTwo)
+        this.specialty = testingPartTwo.specialty;
+        this.favLanguage = testingPartTwo.favLanguage;
+        this.catchPhrase = testingPartTwo.catchPhrase;
+    }
+    demo(subject){
+      return `Today we are learning about ${subject}.`;
+    }
+    grade(student, subject){
+      return `${student.name} receives a perfect score on ${subject}.`;
+    }
+
+}
+
+class Student extends Person{
+    constructor(testingPartThree){
+        super(testingPartThree)
+        this.previousBackground = testingPartThree.previousBackground;
+        this.className = testingPartThree.className;
+        this.favSubjects = testingPartThree.favSubjects;
+    }
+    listsSubjects(){
+      return `${this.favSubjects}.`
```

Comment:
This method should be `a method that logs out all of the student's favoriteSubjects one by one.`.

Given an array, which `favSubjects` is, how might you log each element of the array, one element at a time?